pkgs/applications/science/math/mathematica/default.nix

```diff
@@ -34,139 +42,101 @@
 , xkeyboard_config
 , xorg
 , zlib
+# settings and optional packages
+, cudaSupport ? false
+, cudaPackages ? null
```

Comment:
```suggestion
, cudaPackages
```

Reply:
`cudaPackages` can be left out, and in that case, it should be `null.

Reply:
Overwriting packages to null is undefined behavior. Please use an bool input instead.

Reply:
This is gone after switching to `callPackage`.

Reply:
> `cudaPackages` can be left out, and in that case, it should be `null`.

I support @SuperSandro2000 on this one, if someone wishes to disable cuda support, they can `.override {cudaPackages = null;};` by themselves.